doc/guidelines/development/docker/index.rst

```diff
@@ -4,8 +4,92 @@ Docker Guides
+Name your repo path as "/home/saluser/develop" to reduce confusion.
+Basic idea of giving "--volume" and "-it" is that you are creating container based on the "image", and also making it to connect to your repo, which can be the git cloned repository.
+This way you can make the container and repo to communicate, testing runs in the container, update your repo by copying or movnig the files from the container, and then you can push your local repository to the origin.
+
+The containers created will show up when you click "containers" on the left side of the docker desktop as below. You can stop, or delete the containers with the icons. 
+
+.. image:: ../../../images/containers_indocker.png
+
+When you open the Visual Studio Code and go to the docker icon (whale shape) on the left column, you will see the list of images installed and containers you opened.  
+
+.. figure:: ../../../images/visual_studio_docker.png
+    :width: 400px
+
+Right click while pointing on the container you want to use, and select "attach Visual Studio Code". It will open your container into the new window.
+
+In the local terminal where you just ran the command to open the container, or from the terminal in the visual studio code, run ::
+    
+    bash ~/.setup_dev.sh
+
+.. image:: ../../../images/setup_dev.png
+
+Then go to your repo, which you put into the "--volume", ::
+    
+    cd /home/saluser/develop/(path_to_repos)/ts_standardscripts
+
+then run following commands::
+    
+    setup -r .
+    eups list
+
+You should see something like this::
+
+    ...
+    ts_scriptqueue        tag:current       current setup
+    ts_simactuators       tag:current       current setup
+    ts_standardscripts    LOCAL:/home/saluser/develop/lsst-ts/ts_standardscripts    setup
+    ts_standardscripts    tag:current       current
+    ts_tcpip              tag:current       current setup
+    ... 
+
+This means that you are using your version of the ts_standardscripts instead of the one that comes with the docker image.
+
+In visual studio code, you can also switch branch from the bottom of the window as shown in the image below, click the branch icon, and all the branches will appear on the top box where it was the search box previously.
+
+.. image:: ../../../images/branch_vsc.png
+
+You will NOT see the branches that are up-to-date in git hub because the container is made from the image, and the image is not necesarrily following up with the git hub repository. 
+This is where the repo you called from "--volume" is needed. 
+
+The container created from the image will be under::
+
+    /home/saluser/repos/
+
+while repo is under as you declared in "--volume"::
+
+    /home/saluser/develop/
+
+You can run the files under the /home/saluser/develop/ in terminal (bottom red box in the following image) or edit them opening the files under the /home/saluser/develop/ directory. 
```

Comment:
I don't understand what you mean here. What you describe below is basically the way we clone a repository, which is not within the scope of these guidelines. I think you can keep it, but maybe say something to warn the user that the environment is ready to work on it and that you can clone the repo, etc (or start your own repository).

Reply:
Ah, sorry, I had wrong image put under the different description